src/components/Board/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import App from './App'
+import List from './list'
+import Card from './card'
+import Button from './Button'
+
+/**
+~~~~~~~~~~~ List Structure ~~~~~~~~~~~
+list: [{
+    title: "",
+    cards: [{
+        id: "",
+        text: "",
+        moveForward: () => {},
+        moveBackward: () => {},
+    }],
+}],
+}
+**/
+
+export default class Board extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {list: []}
+    }
+
+    render() {
+        return (
+            <h1> {this.props.name} </h1>
+            <div className='board-wrap'>
+                this.state.list.map((el) => (
```

Comment:
one caveat about this is that it'll immediately need to get re-rendered after the data transformation is finished. i'd recommend adding another flag in the state that keeps track of whether or not the transformation is finished, and displaying a loading screen until it has. don't need to do in this merge, but something to do in the future